modules/waypoints_to_commands.py

```diff
@@ -29,8 +31,9 @@ def waypoints_to_commands(waypoints: "list[tuple[float, float]]",
     """
     dronekit_command_list = []
 
-    for waypoint in waypoints:
-        lat, lng = waypoint
+    for point in waypoints:
+        lat = point.latitude
+        lng = point.longitude
```

Comment:
Could remove these intermediate variables and use them directly in the constructor.